src/sendGCM.js

```diff
@@ -27,6 +27,7 @@ const sendChunk = (GCMSender, registrationTokens, message, retries) => new Promi
                     regIndex += 1;
                     return {
                         messageId: value.message_id,
+                        oldRegId: (value.registration_id !== regToken) ? regToken : null,
```

Comment:
Semantically it could make sense to always set this to `regToken` because there is always a "old registration id", namely `regToken`. Applications that use this, could simply do the old vs. new check in their code, if required.

Reply:
Yes, from users' perspective it would be more clear to just give them back the 'regToken'. But I'll rename the parameter to 'inputRegId', as 'oldRegId' subtly implies that there's a new registration id generated, which may lead to ambiguity. Thanks for pointing out a better solution.